packages/flutter/lib/src/material/list_item.dart

```diff
@@ -36,11 +37,17 @@ class ListItem extends StatelessComponent {
   final Widget right;
   final bool isThreeLine;
   final bool isDense;
```

Comment:
It would be nice to just have one layoutStyle parameter. It might end up looking like:

ListItemLayoutStyle {oneLine, twoLine, threeLine, denseOneLine, denseTwoLine, denseThreeLine}